lte/gateway/deploy/agw_install_ubuntu.sh

```diff
@@ -72,13 +73,16 @@ if [[ $1 != "$CLOUD_INSTALL" ]] && ( [[ ! $INTERFACES == *'eth0'*  ]] || [[ ! $I
 
   # interface config
   apt install -y ifupdown net-tools
+  mkdir -p "$INTERFACE_DIR"
+  echo "source-directory $INTERFACE_DIR" > /etc/network/interfaces
+
   echo "auto eth0
-  iface eth0 inet dhcp" > /etc/network/interfaces.d/eth0
+  iface eth0 inet dhcp" > "$INTERFACE_DIR"/eth0
```

Comment:
INTERFACE_DIR already contains trailing `/`? would this not result in double backlash?

Reply:
removed it.